example/src/Configuration.js

```diff
@@ -12,22 +12,54 @@ export const CHANNEL = Platform.select({
 })();
 
 export const DEFAULT_CONFIGURATION = {
```

Comment:
The idea of `DEFAULT_CONFIGURATION` is to provide a plain list of constants and properties and feature-toggles for App's "Configuration".
All necessary payment configurations are expected on `AdyenCheckout` directly - this will act as code sample, so it is easy to copy-past